pkg/cmd/server/api/types.go

```diff
@@ -132,15 +132,21 @@ type NodeConfig struct {
 	IPTablesSyncPeriod string
 
 	// VolumeConfig contains options for configuring volumes on the node.
-	VolumeConfig VolumeConfig
+	VolumeConfig NodeVolumeConfig
 }
 
-// VolumeConfig contains options for configuring volumes on the node.
-type VolumeConfig struct {
+// NodeVolumeConfig contains options for configuring volumes on the node.
+type NodeVolumeConfig struct {
 	// LocalQuota contains options for controlling local volume quota on the node.
 	LocalQuota LocalQuota
 }
 
+// MasterVolumeConfig contains options for configuring volumes on the node.
+type MasterVolumeConfig struct {
+	// EnableDynamicProvisioning is a boolean that toggles the dynamic provisioning feature on/off
+	EnableDynamicProvisioning bool
```

Comment:
I don't really want a one-off flag for a particular controller, I'd rather work to make the `controllers` config param effective for disabling a subset of controllers